src/api/app/models/bs_request_action_maintenance_release.rb

```diff
@@ -116,18 +116,13 @@ def set_acceptinfo(ai)
     self.bs_request_action_accept_info = BsRequestActionAcceptInfo.create(ai)
   end
 
-  def create_post_permissions_hook(opts)
+  def create_post_permissions_hook
     object = nil
```

Comment:
```suggestion
```

Reply:
Please. remove this assignment. It's doing nothing and it's preventing the checks from going green.